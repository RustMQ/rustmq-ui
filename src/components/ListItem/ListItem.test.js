import React from "react";
import LinkItem from "./ListItem";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";

it("renders correctly", () => {
  const queue = { id: 1, name: "Test pull", type: "pull", size: 38 };
  const tree = renderer
    .create(
      <MemoryRouter>
        <div>
          <LinkItem queue={queue} />
        </div>
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
