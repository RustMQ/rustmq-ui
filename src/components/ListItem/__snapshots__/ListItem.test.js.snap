// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <li>
    <a
      href="/queue/1"
      onClick={[Function]}
    >
      [Size: 
      38
      ] 
      Test pull
       (
      pull
      )
    </a>
  </li>
</div>
`;
